static/css/styles.css

```
@@ -583,6 +583,24 @@ h2 {
 */
 
 
+.other-products-heading {
+  margin: 40px 10px 10px 10px;
```

I think a larger margin on the right would help balance out the visual weight of sidebar with the arrows `>`above it. 

```suggestion
  margin: 40px 25px 10px 10px;
```